docs/app/pages/css/sections/labels/labels/labels.component.html

```diff
@@ -1,58 +1,126 @@
 <p>General Labels</p>
-<span class="label label-primary m-r-sm">Label Primary</span>
-<span class="label label-accent m-r-sm">Label Accent</span>
-<span class="label label-grey4 m-r-sm">Label Grey4</span>
+<span class="label label-primary m-r-sm">Featured</span>
+<span class="label label-accent m-r-sm">Recently Added</span>
+<span class="label label-grey4 m-r-sm">On Hold</span>
 
 <p class="m-t-md">Outline Labels</p>
-<span class="label label-outline-alternate2 m-r-sm">Outline Label Alternative2</span>
-<span class="label label-outline-alternate3 m-r-sm">Outline Label Alternative3</span>
-<span class="label label-outline-grey4">Outline Label Grey4</span>
+<span class="label label-outline-vibrant2 m-r-sm">Contact</span>
+<span class="label label-outline-vibrant1 m-r-sm">Company</span>
+<span class="label label-outline-alternate1">Location</span>
 
 <p class="m-t-md">Status Labels</p>
-<span class="label label-ok m-r-sm">Label OK</span>
-<span class="label label-warning m-r-sm">Label Warning</span>
-<span class="label label-critical">Label Critical</span>
+<span class="label label-ok m-r-sm">Completed</span>
+<span class="label label-warning m-r-sm">Cancelled</span>
+<span class="label label-critical">Failed</span>
```

Comment:
The example code and codepen are not up to date with this change - see the snippets subdirectory.